build.gradle

```diff
@@ -1,14 +1,23 @@
 plugins {
-    id 'pl.allegro.tech.build.axion-release' version '1.16.1'
+    alias(libs.plugins.axionRelease)
+    alias(libs.plugins.nexusPublish)
+    id 'java'
 }
 
 group 'com.rundeck'
 
 ext.rundeckPluginVersion = '1.2'
+ext.publishName = "Git Plugin ${project.version}"
+ext.githubSlug = 'rundeck-plugins/git-plugin'
```

Comment:
you will need to add `developers` content here, maven central will reject it unless it defines developer info.  ref: https://github.com/rundeck-plugins/slack-incoming-webhook-plugin/blob/main/build.gradle#L13-L15